server/database/models/conversation.js

```diff
@@ -0,0 +1,21 @@
+let uuid = require('uuid');
+
+class Conversation {
+    constructor(creator, type, name = null) {
+        this.id = uuid.v4();
+        if (name) {
+            this.name = name;
+        }
```

Comment:
There is no point of getting as default name equals null and checking it. you can get as default undefined and than assign it to name. name as default is undefined..